crypto/bio/bss_dgram.c

```diff
@@ -670,7 +671,7 @@ static long dgram_ctrl(BIO *b, int cmd, long num, void *ptr)
                  * we assume that the transport protocol is UDP and no IP
                  * options are used.
                  */
```

Comment:
This comment can be deleted since it applies to the implementation in `dgram_get_mtu_overhead()`